Package.swift

```diff
@@ -27,3 +27,11 @@ let package = Package(
         .testTarget(name: "MQTTNIOTests", dependencies: ["MQTTNIO"]),
     ]
 )
+
+
+#if swift(>=5.6)
+  // Add the documentation compiler plugin if possible
+  package.dependencies.append(
+    .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0")
+  )
+#endif
```

Comment:
I've been building docc outside of the package, this is a nice way to include it as a plugin.